install.yml

```diff
@@ -6,23 +6,19 @@
   become: True
 
   tasks:
-
     - name: "Install required packages"
       apt:
         name: "{{ item }}"
         state: present
       with_items:
         - "build-essential"
-        - "python-dev"
+        - "python3-dev"
 
     # we need to download and install manually the lib because the pypi package is not up to date.
-    # see https://github.com/adafruit/Adafruit_Python_DHT/issues/77
-    - name: "Clone Adafruit python lib project"
-      git:
-        repo: "https://github.com/adafruit/Adafruit_Python_DHT.git"
-        dest: "/tmp/Adafruit_Python_DHT"
+    - name: "install Adafruit_dht for dependence"
+      shell: "sudo pip3 Adafruit_dht"
 
     - name: "Install Adafruit python lib"
-      shell: "python setup.py install"
+      shell: "sudo python3 setup.py install"
```

Comment:
If the pip command as been used. The lib is installed. So why keeping this task?